### PR TITLE
add location threshold

### PR DESCRIPTION
**Pull request by loinguyenhdwebsoft**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -154,14 +162,19 @@ const TinderCard = React.forwardRef(({ flickOnSwipe = true, children, onSwipe, o
...
 
-  const handleSwipeReleased = React.useCallback(async (element, speed) => {
+  const handleSwipeReleased = React.useCallback(async (element, speed, lastLocation) => {
     if (swipeAlreadyReleased.current) { return }
     swipeAlreadyReleased.current = true
 
     // Check if this is a swipe
-    if (Math.abs(speed.x) > settings.swipeThreshold || Math.abs(speed.y) > settings.swipeThreshold) {
-      const dir = getSwipeDirection(speed)
+    let checkMore = false
```
Comment (jayashe) [inline index.js]:
```suggestion
    let swipeDistanceThresholdMet = false
```

a little more readable

**Review by jayashe — CHANGES_REQUESTED** [review]

**Review by jayashe — APPROVED** [review]

**Comment by jayashe** [comment]
Can you merge this, then update the react-tinder-card dependency in the style-quiz-frontend repo to point to this?

**Comment by loinguyenhdwebsoft** [comment]
You want `react-tinder-card` like `domain-entity`, right? We will be using the npm registry

**Comment by jayashe** [comment]
@loinguyenhdwebsoft yeah you should be able to reference this repo directly

**Pull request merged**